### PR TITLE
fix(pm2): update scripts and add dev deps

### PR DESCRIPTION
**Pull request by varunzxzx**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Removed some unnecessary imports. Changed es5 `var` to es6 `const`. Also, removed unnecessarily quoted property.

### CODE REVIEW

**Comment by tchaffee** [comment]
I noticed that `package.json` doesn't have pm2 listed as a dependency. So this script will fail until you install it manually. This PR would be a good place to fix that.

**Comment by varunzxzx** [comment]
@tchaffee I think we need to install it globally.

**Comment by tchaffee** [comment]
That might make sense. But *why* install it globally? If there is a good reason behind that decision, then I'd like to see the reason documented. Otherwise, I avoid installing packages globally whenever possible.

**Comment by varunzxzx** [comment]
Ohh Sorry! I just checked that we are not running pm2 using the command line. I will add it to the package.json. Thanks for the suggestion @tchaffee

File: package.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
     "passport-local": "^1.0.0",
     "passport-oauth": "^1.0.0",
     "passport-twitter": "^1.0.3",
+    "pm2": "^2.10.2",
```
Comment (raisedadead) [inline package.json]: We do not use this flow in development, hence adding this is not needed. However if you are still interested, please move it to dev deps.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (varunzxzx) [reply package.json]: Ok. I'll move it to dev dependencies.

**Review by raisedadead — APPROVED** [review]

**Comment by tchaffee** [comment]
Maybe too late now, but isn't pm2 a production dependency rather than a dev dependency? The `pm2Start.js` script is used for production, right?

**Pull request merged**